blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/RunContainerImpl.java

```diff
@@ -59,6 +80,23 @@ public BlueRun get(String name) {
         return  AbstractRunImpl.getBlueRun(run, pipeline);
     }
 
+    /**
+     * Finds a run in the queue based on the expectedBuildNumber, returns the queue id if found
+     */
+    private String findRunInQueue(String name) {
```

Comment:
Again ... the name seems confusing to me. "findX" would suggest to me that the function is going to return an X, but the pattern in this PR seems to be the other way around i.e. you are supplying X and returning something else.